src/examples/fetch_env_vars.yml

```diff
@@ -12,6 +12,10 @@ usage:
     ephemeral-e2e-tests:
       executor:
         name: cypress/browsers-chrome77
+      parameters:
+        api-token:
+          type: env_var_name
+          default: SHIPYARD_API_TOKEN
```

Comment:
I assume that this is in response to this comment on the issue: 

> - Please add the env_var_name parameter to your API token for documentation purposes. You can find an example in our [Orb Authoring Best Practices](https://circleci.com/docs/2.0/orbs-best-practices/?section=configuration) under the "Commands" section

I _think_ he means that the __command__ should require that parameter. Here you have it in our **example** only. The example would show that the __command__ requires the parameter. Looking at the [orb for Node](https://circleci.com/developer/orbs/orb/circleci/node), they have parameters in their commands, and the orb page automatically highlights them in a table. (I am not 100% sure if it's needed in the example)

The `shipyard/fetch-shipyard-env` command lives [here](https://github.com/shipyardbuild/circleci-orb/blob/master/src/commands/fetch-shipyard-env.yml). I referred to the documentation on parameters, in [a different location than the best practices](https://circleci.com/docs/2.0/reusing-config/#using-the-parameters-declaration).

Note that this means the command & `.sh` script should probably take in the API token as input, instead of inferring it from the environment.

Ofc, feel free to ask to chat with me if you like.

Reply:
Ah yes, I was confused about it but that makes way more sense. I added it to the command and formatted it according to the documentation, but let me know if I missed anything. I also renamed the example 👍